ui/src/components/uploader/QUploader.json

```diff
@@ -152,5 +152,53 @@
         }
       }
     }
+  },
+
+  "fields": {
```

Comment:
If we don't merge in https://github.com/quasarframework/quasar/pull/14241 as I suggested to avoid, these definitions need to be moved into scoped slots params